nuspec/nuget/Cake.SendGrid.nuspec

```diff
@@ -3,15 +3,19 @@
   <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <id>Cake.SendGrid</id>
     <version>0.0.0</version>
-    <authors>gep13</authors>
-    <owners>gep13</owners>
+    <authors>jericho</authors>
+    <owners>jericho</owners>
     <description>Cake SendGrid AddIn.</description>
-    <summary>Cake AddIn that extends Cake with ability to execute the gem command line utility.</summary>
+    <summary>Cake AddIn that extends Cake with ability to send emails via SendGrid.</summary>
     <licenseUrl>https://github.com/cake-contrib/Cake.SendGrid/blob/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/cake-contrib/Cake.SendGrid/</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <copyright>Copyright (c) Cake Contributions 2016 - Present</copyright>
     <tags>Cake, Script, Build, SendGrid</tags>
+    <dependencies>
+      <dependency id="Newtonsoft.Json" version="9.0.1" />
```

Comment:
that's really unfortunate. This means that I need to find out all the DLLs needed by Newtonsoft.Json and any of it's own dependencies. I'll try to figure it out